.github/workflows/assets-validation.yaml

```diff
@@ -47,12 +51,28 @@ jobs:
         uses: actions/setup-python@v4
         with:
           python-version: '>=3.8'
-    
+
+      - name: print
+        run: echo $client_id $tenant_id $subscription_id
+        env: 
+          client_id: ${{ secrets.AZURE_CLIENT_ID }}
+          tenant_id: ${{ secrets.AZURE_TENANT_ID }}
+          subscription_id: ${{ secrets.AZURE_SUBSCRIPTION_ID }}
+  
+      - name: Log in to Azure
```

Comment:
Could you put this log in section after install dependencies please?

Reply:
removed